pkg/controllers/dashboardapi/settings/settings_test.go

```diff
@@ -28,6 +28,7 @@ type testCase struct {
 }
 
 func TestSetAll(t *testing.T) {
+	t.Skip()
```

Comment:
@rafaelbreno can you make note of which tests are being skipped to address later?

Reply:
Yep, will do it.

I'm just going to wait after squashed the commits, so I can link them easily.